debian/mariadb-server-10.5.mysql-server.logrotate

```diff
@@ -11,7 +11,7 @@
 	sharedscripts
 	postrotate
           test -x /usr/bin/mysqladmin || exit 0
-          if [ -f `my_print_defaults --mysqld | grep -oP "pid-file=\K[^$]+"` ]; then
+          if [ -f `my_print_defaults --mysqld | grep -m 1 -oP "pid-file=\K.+$"` ]; then
             # If this fails, check debian.conf!
```

Comment:
Suggest using something like: https://github.com/MariaDB/server/pull/934#discussion_r340256147

Reply:
Dropped this commit from this PR, let the other PR fix this instead once and for all.